python/GafferUI/Image.py

```diff
@@ -61,6 +61,8 @@ def __init__( self, imagePrimitiveOrFileName, **kw ) :
 
 		if isinstance( imagePrimitiveOrFileName, str ) :
 			pixmap = self._qtPixmapFromFile( str( imagePrimitiveOrFileName ) )
+		elif isinstance( imagePrimitiveOrFileName, QtGui.QPixmap ) :
+			pixmap = imagePrimitiveOrFileName
```

Comment:
> this has required generating a GafferUI.Image from a QPixmap - maybe this is too much coupling with Qt?

Yeah, we don't want to be passing around Qt objects in the public API like this.